src/native/etw.rs

```diff
@@ -342,3 +351,264 @@ impl super::EventWriter<Provider> for Provider {
         });
     }
 }
+
+struct CommonSchemaPartCBuilder<'a> {
```

Comment:
This can probably be moved to a common location to be shared between etw and user_events.